pkg/pipeline/extract/conntrack/conntrack.go

```diff
@@ -61,6 +61,16 @@ func (ct *conntrackImpl) Extract(flowLogs []config.GenericMap) []config.GenericM
 
 	var outputRecords []config.GenericMap
 	for _, fl := range flowLogs {
+		if !fl.IsValidProtocol() {
+			log.Debugf("skipping layer2 protocols flow log %v", fl)
+			ct.metrics.inputRecords.WithLabelValues("discarded").Inc()
+			continue
+		}
+		if fl.IsNoneTransportProtocol() {
+			log.Debugf("skipping none transport protocols (ICMPv4/6) flow log %v", fl)
```

Comment:
Perhaps we should state this condition the other way around?
```golang
if !IsTransportProtocol() {
```
and say that conntrack supports only TCP and UDP